SocialFilter/sections/specific.txt

```diff
@@ -2130,7 +2129,7 @@ mapsofworld.com##.social_icon
 navbharattimes.indiatimes.com##.wdt_social_share > .social-cell:not(.comments)
 edh.tw##.line-friend
 edh.tw##.linefriend_article
-trybotics.com##button[aria-label="Share"]
+google.com,tunecore.co.jp,trybotics.com##button[aria-label="Share"]
```

Comment:
Need a different rule `Share` word is translated 
![image](https://user-images.githubusercontent.com/8361299/164203845-0ad19cb6-b21a-4506-9b3f-73c0babbbcb1.png)

For desctop version another rule is required.